gammapy/datasets/make.py

```diff
@@ -64,3 +71,121 @@ def sigma_energy_theta(energy, theta, sigma):
                                        zenith=zenith_hi)
 
     return psf
+
+
+def make_test_observation_table(observatory, n_obs):
+    """Generate an observation table.
+
+    For the moment, only random observation tables are created.
+
+    Parameters
+    ----------
+    observatory : string
+    	name of the observatory
```

Comment:
I added a reference, without leaving a TODO, because chances are that at some point the list will appear in the doc, and we forget to remove the TODO.